Gopkg.toml

```diff
@@ -29,29 +7,25 @@
   name = "github.com/golang/glog"
 
 [[constraint]]
+  name = "github.com/kubernetes-incubator/external-storage"
   branch = "master"
-  name = "github.com/golang/protobuf"
 
 [[constraint]]
-  branch = "master"
-  name = "github.com/kubernetes-incubator/external-storage"
+  name = "google.golang.org/grpc"
+  version = "1.10.0"
 
 [[constraint]]
   branch = "master"
-  name = "golang.org/x/net"
+  name = "k8s.io/kubernetes"
 
 [[constraint]]
-  name = "google.golang.org/grpc"
-  version = "1.7.2"
+  branch = "master"
```

Comment:
we should try to track a specific version if possible

Reply:
done